libnavigator/proguard-rules.pro

```diff
@@ -0,0 +1,5 @@
+-keep class com.mapbox.navigation.navigator.MapboxNativeNavigatorImpl { *; }
```

Comment:
Why do we need to keep the navigator? It's always strongly referenced.

Reply:
think it was an experimental rule, when I tested minifying. Remove it, great catch 👍